hc/api/tests/test_list_checks.py

```diff
@@ -33,15 +33,28 @@ def get(self):
 
     def test_it_works(self):
         r = self.get()
-        ### Assert the response status code
-
         doc = r.json()
         self.assertTrue("checks" in doc)
 
         checks = {check["name"]: check for check in doc["checks"]}
-        ### Assert the expected length of checks
-        ### Assert the checks Alice 1 and Alice 2's timeout, grace, ping_url, status,
-        ### last_ping, n_pings and pause_url
+        assert len(checks) == 2
+        a1_pause_url = "http://localhost:8000/api/v1/checks/"+str(self.a1.code)+"/pause"
+        self.assertEqual(checks["Alice 1"]["timeout"], self.a1.timeout.total_seconds())
+        self.assertEqual(checks["Alice 1"]["grace"], self.a1.grace.total_seconds())
+        self.assertEqual(checks["Alice 1"]["ping_url"], self.a1.url())
+        self.assertEqual(checks["Alice 1"]["status"], self.a1.status)
+        self.assertEqual(checks["Alice 1"]["last_ping"], self.a1.last_ping.isoformat())
+        self.assertEqual(checks["Alice 1"]["n_pings"], self.a1.n_pings)
+        self.assertEqual(checks["Alice 1"]["pause_url"], a1_pause_url)
+
+        a2_pause_url = "http://localhost:8000/api/v1/checks/"+str(self.a2.code)+"/pause"
```

Comment:
Will there ever be a point where this URL changes? ie change from locslhost url to maybe Heroklu Url or better yet, the port number.... How will this test be affected? Find a way to maybe specify the URL as an environment variable.

Reply:
I have created a variable to the URL.